google/cloud/spanner_dbapi/parser.py

```diff
@@ -147,6 +161,19 @@ def parse_values(stmt):
 
 
 def expect(word, token):
+    """Parse the given expression recursively.
+
+    :type word: str
+    :param word: A string expression.
+
+    :type token: str
+    :param token: An expression token.
+
+    :rtype: (str, Any)
+    :returns: A tuple of the rest expression string and tree of already parsed
```

Comment:
`A tuple containing the rest of the expression string and the tree of the already parsed`